tests/output/ConsoleInteractionTest.py

```diff
@@ -849,3 +849,16 @@ def test_empty_list(self):
                                 None,
                                 None,
                                 None)
+
+    def test_formatted_string(self):
+        self.section.append(Setting(key='format', value='{source_lines}'))
+        affected_code = (SourceRange.from_values('file', 2, end_line=2),)
+        with retrieve_stdout() as stdout:
+            print_results_formatted(
+                self.logger,
+                self.section,
+                [Result('SpaceConsistencyBear', message='msg',
+                        affected_code=affected_code)],
+                {abspath('file'): ('def fun():\n', '    pass  \n')})
+            self.assertRegex(stdout.getvalue(),
+                             "['    pass  \\n']")
```

Comment:
are we sure we want them to be printed as a list and not the list joined with a newline?